mne/io/meas_info.py

```diff
@@ -347,6 +348,8 @@ def __repr__(self):
         non_empty = 0
         for k, v in self.items():
             if k in ['bads', 'ch_names']:
+                if not isinstance(v, list):
+                    v = list(v)
```

Comment:
@wmvanvliet to have a clean list presentation that does not expose a private class